build.gradle.kts

```diff
@@ -1,38 +1,10 @@
-
-buildscript {
-    repositories {
-        google()
-        mavenCentral()
-        maven("https://dl.bintray.com/jetbrains/kotlin-native-dependencies")
-        maven("https://plugins.gradle.org/m2/")
-        maven("https://oss.sonatype.org/content/repositories/snapshots")
-        jcenter()
-    }
-
-    dependencies {
-        classpath(Plugins.kotlin)
-        classpath(Plugins.dokka)
-        classpath(Plugins.android)
-        classpath(Plugins.atomicFu)
-    }
-}
-
 plugins {
-    id("de.fayard.buildSrcVersions") version "0.4.2"
+  id("com.diffplug.spotless")
 }
 
-allprojects {
-    repositories {
-        google()
-        jcenter()
-        maven("https://kotlin.bintray.com/kotlinx")
-        maven("https://oss.sonatype.org/content/repositories/snapshots")
-        mavenCentral()
-    }
-
-    group = project.properties["GROUP"]!!
-    version = project.properties["VERSION_NAME"]!!
-    if (hasProperty("SNAPSHOT") || System.getenv("SNAPSHOT") != null) {
-        version = "$version-SNAPSHOT"
-    }
+gradleEnterprise {
```

Comment:
I've been away from gradle for awhile - what's the deal with `gradleEnterprise`?

Reply:
This basically allows build scans in CI which makes it a lot easier to debug issues